games-arcade/ecwolf/ecwolf-1.4.1.recipe

```diff
@@ -0,0 +1,105 @@
+SUMMARY="Enhanced Wolfenstein 3D source port"
+DESCRIPTION="ECWolf is an advanced source port for Wolfenstein 3D, \
+Spear of Destiny and Super 3D Noah's Ark based off of the Wolf4SDL \
+code base. ECWolf aims to provide a wide array of mod editing \
+capabilities without the need to modify the source code.
+
+For players, ECWolf provides many useful features such as better \
+control binding, support for wide screen resolutions, and unlimited \
+save slots. ECWolf allows you to keep all of your game data \
+in one directory as one binary plays all supported games."
+HOMEPAGE="https://maniacsvault.net/ecwolf/"
+COPYRIGHT="
+	2004-2024 ECWolf team
+	1992 id Software
+	"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://maniacsvault.net/ecwolf/files/ecwolf/1.x/ecwolf-${portVersion}-src.tar.xz"
+CHECKSUM_SHA256="8ebd495d2806c9d0e736656970e736730a005d3b43c7f5729f52c812b22f9e2d"
+SOURCE_DIR="ecwolf-${portVersion}-src"
+PATCHES="ecwolf-$portVersion.patchset"
+ADDITIONAL_FILES="ecwolf.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	ecwolf$secondaryArchSuffix = $portVersion
+	app:ecwolf = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libSDL2_mixer_2.0$secondaryArchSuffix
+	lib:libSDL2_net_2.0$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libflac$secondaryArchSuffix
+	devel:libfluidsynth$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libmodplug$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libopus$secondaryArchSuffix
+	devel:libopusfile$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libSDL2_mixer_2.0$secondaryArchSuffix
+	devel:libSDL2_net_2.0$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	export LDFLAGS="-lbsd -lnetwork"
+
+	mkdir -p build && cd build
+
+	# Set up CMake paths
+	sed -i -e "s|@APPSDIR@|$appsDir|" \
+		-i -e "s|@DATADIR@|$dataDir|" \
+		-i -e "s|@DOCDIR@|$docDir|" \
+		-i -e "s|@MANDIR@|$manDir|" \
+		$sourceDir/CMakeLists.txt
```

Comment:
Using `$cmakeDirArgs` instead of this should work fine, I think. This was already using `GNUInstallDirs`, so it shouldn't require patching the cmake file.

Reply:
Put some actual effort into the patch, also accidentally fixed the man documentation :)